ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcMethod.java

```diff
@@ -94,6 +94,7 @@
   ETH_NEW_PENDING_TRANSACTION_FILTER("eth_newPendingTransactionFilter"),
   ETH_PROTOCOL_VERSION("eth_protocolVersion"),
   ETH_SEND_RAW_TRANSACTION("eth_sendRawTransaction"),
+  ETH_BATCH_RAW_TRANSACTION("eth_batchSendRawTransaction"),
```

Comment:
Unless this is a standard (or standards track) API I think we should either take it out of the `eth_` namespace (`debug_` or an new one) or make it obviously bogus, like  `eth_x_batchSendRawTransaction`

Reply:
Removed from this PR 
cc - @abdelhamidbakhta